charts/zqd/templates/deployment.yaml

```diff
@@ -84,7 +84,9 @@ spec:
             {{- toYaml .Values.resources | nindent 12 }}
           volumeMounts:
             - mountPath: /spaces
-              name: spaces-volume     
+              name: spaces-volume
+            - mountPath: /tmp
```

Comment:
Did you need to add this because there was no `/tmp` present on the filesystem? Or because you wanted to ensure the directory was empty on launch?

Reply:
The /tmp directory was not present. Making /tmp a emptyDir{} will allow files written to /tmp to go to a ramdisk which will have the same lifetime as the container.

Reply:
@marktwallace: A RAM disk isn't going to work very well for an S3 object larger than RAM.